src/components/InteractionLayer/index.js

```diff
@@ -566,13 +566,7 @@ class InteractionLayer extends React.Component {
         }
       }
       const color = scaledArea.color || (s ? s.color : null);
-      return (
-        <Area
-          key={`${scaledArea.id || ''}-${scaledArea.seriesId || ''}`}
```

Comment:
I'm a little concerned that the seriesId portion here was added for a reason, and might not want to be removed... @evancharlton ?

Reply:
Merged but because we drop this array in the fragment with a bunch of other arrays of elements at the same level, the area id isn't necessarily going to be unique as it could clash with the annotation ids...

Not sure about why the seriesId might be needed but I'll fix the above issue.

Reply:
https://github.com/cognitedata/griff-react/pull/337